tests/acceptance/host/profile-test.js

```diff
@@ -47,11 +47,13 @@ module('Acceptance | host/profile', function(hooks) {
     // Greeting
     assert.equal(currentRouteName(), 'auth.host.greeting');
     await greetingPage.profilePic.setImage(new File([ imageBlob ], 'foo.jpg', { type: 'image/jpeg' }));
+    await greetingPage.profileName.fillIn('The Bluth family');
```

Comment:
Hi Ben, just curious about what is the greetingPage.profileName.fillIn('The Bluth family') used here. Is it the testing usage?